sdk/rust/examples/backend/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+workspace = { members = ["axum-backend"] }
+[package]
+name = "backend"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { version = "4.5.4", features = ["derive"] }
+dagger-sdk = "0.9.8"
```

Comment:
We should depend on the version from the workspace. That way we can actually verify the examples work using the current version. Can be done with 

```toml
dagger-sdk = {path = "../../crates/dagger-sdk"} # may not be the correct path fyi :)
```

Or we can set the dagger-sdk in the workspace and use that here

```toml
# Root Cargo.toml
[workspace.dependencies]
dagger-sdk = {path = "crates/dagger-sdk"}

# ---
# file: examples/backend/Cargo.toml
[dependencies]
dagger-sdk.workspace = true
```

Reply:
We in general if we can use workspace versions while still keeping the examples clear and easy to use that would be a boon, as we then don't have to maintain many versions of the same packages.